code/modules/mob/mob_helpers.dm

```diff
@@ -52,7 +52,13 @@
 	return 0
 
 /proc/isvaurca(A)
-	if(istype(A, /mob/living/carbon/human) && (A:get_species() == "Vaurca"))
+	if(istype(A, /mob/living/carbon/human) && (A:get_species() == "Vaurca Worker"))

```

Comment:
This can be written more efficiently.
```
    if(istype(A, /mob/living/carbon/human))
        switch(A:get_species())
            if ("Vaurca Worker")
              return 1
            ...blablabla

    return 0
```